frontend/css/explore.less

```diff
@@ -0,0 +1,61 @@
+.explore-card-container {
+  width: 100%;
+  float: left;
+  padding: 0 15px;
+
+  @media (min-width: 600px) {
+    width: 50%;
+  }
+  @media (min-width: 900px) {
+    width: 33%;
+  }
+}
```

Comment:
For posterity, I think if we ever revisit this, we should look into CSS grids instead, which seem to be an ideal candidate for this type of layout.
Not something we should do right now though !